README.md

```diff
@@ -61,7 +61,3 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
```

Comment:
Please, why did you delete the License session?

Reply:
> Please, why did you delete the License session?

- [ ] Would you please provide at least two comment reviews requesting changes and giving reasons as to `why?` the changes are requested

Reply:
Please be conscious of the emoji you use so as not to allow the student to get the wrong massage thinking you are mocking him/her, using the :+1:  is safer.